mujoco_warp/_src/jax_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2025 The Newton Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import os
+
+import jax
+import warp as wp
+from jax import numpy as jp
+from warp.jax_experimental.ffi import jax_callable
+
+import mujoco_warp as mjwarp
+from mujoco_warp._src.test_util import fixture
+
+if jax.default_backend() == "gpu":
```

Comment:
does this run when you run pytest?

Can we structure this as an absltest?

Also, can you leave a comment somewhere so that it's really clear that this is temporary and that it will be removed after we land MJX:Warp?

Reply:
yes this will run when we run pytest

the test is now structured as an absltest

added a todo